django-stubs/db/models/base.pyi

```diff
@@ -27,6 +27,7 @@ class Model(metaclass=ModelBase):
     _state: ModelState
     def __init__(self: _Self, *args, **kwargs) -> None: ...
     @classmethod
```

Comment:
You are missing `@classmethod` decorator for `from_db` now 🙂

Reply:
Thanks